modules/nodes-adding-bare-metal-host-to-cluster-using-cli.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-cli_{context}"]
+= Adding a bare metal host to the cluster and provisioning a bare metal node using the CLI
+
+You can add bare metal hosts to the cluster and then provision them as bare metal nodes using the CLI.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a bare metal host for use in the cluster by completing the following actions:
+.. Create the following `BareMetalHost` Custom Resource (CR) that defines the `Secret` credentials and the required cluster node namespace, for example `openshift-worker`. Save the YAML in the `openshift-worker-0-3-namespace.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-0-3-bmc-secret
+type: Opaque
+data:
+  username: <secret-username>
+  password: <secret-password>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker-0-3
+spec:
+  online: true
+  bmc:
+    address: <BMC address>
+    credentialsName: openshift-worker-0-3-bmc-secret
+    disableCertificateVerification: True
+    username: <username>
+    password: <password>
+  bootMACAddress: <host boot MAC address>
+  hardwareProfile: unknown
+----
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f openshift-worker-0-3-namespace.yaml -n openshift-machine-api
+----
+.. Check the status of the new bare metal host instance and ensure it is available in the cluster:
++
+[source,terminal]
+----
+$ oc get bmh -A
+----
++
+.Example output
++
+[source,terminal]
+----
+NAMESPACE               NAME                   STATE                    CONSUMER                                  ONLINE   ERROR
+openshift-machine-api   openshift-master-0-0   externally provisioned   ocp-edge-cluster-0-sfljt-master-0         true
+openshift-machine-api   openshift-worker-0-3   provisioned              ocp-edge-cluster-0-sfljt-worker-0-sv86d   true
+----
+. Using the relevant machine set, scale the number of nodes in the cluster to include the new bare metal host.
```

Comment:
```suggestion
. Using the relevant machine set, scale the number of machine replicas it creates so that a new machine is available to bind to the bare metal host to provision it into a node.
```

Rough, but something to this effect, which explains how this action will result in a new node.

Reply:
What about: 

`. Using the relevant machine set, scale the number of machine replicas to accommodate the new bare metal host and provision it as a node.`